### PR TITLE
Add map with cities

### PR DESCRIPTION
**Pull request by reponche**

### CODE REVIEW

File: weatherbot/main.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def main():
     puller = State("./puller_state")
     while True:
         updates = get_updates()
-        if updates['ok'] is True:
+        if updates['result'] == [] or updates["ok"] is False:
```
Comment (chuwy) [inline weatherbot/main.py]: It's very likely that response won't contain `result` key at all if error happened, so I'd leave only `updates["ok"] is False` check.

File: weatherbot/map.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from weatherbot.weather import get_weather
+
+
+def get_coords(city):
+    coords_list = []
+    response = get_weather(city)
+    coords = response["coord"]
+    coords_list.append(coords['lat'])
+    coords_list.append(coords['lon'])
+    return coords_list
```
Comment (chuwy) [inline weatherbot/map.py]: I'd rewrite it as `return [coords['lat'], coords['lon']]`

File: weatherbot/map.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+m = folium.Map(location=[0, 0], zoom_start=2)
```
Comment (chuwy) [inline weatherbot/map.py]: No idea what you're doing here.

File: puller_state [hunk 4]
```diff
@@ -1 +1 @@
-490692396
\ No newline at end of file
+490692413
```
Comment (chuwy) [inline puller_state]: I thought we agreed to delete this file.

Comment (reponche) [reply puller_state]: I put this file to gitignore.

File: weatherbot/map.py (hunk 3 above)
Comment (reponche) [reply weatherbot/map.py]: This command creates the map.

File: weatherbot/map.py (hunk 2 above)
Comment (reponche) [reply weatherbot/map.py]: Okay, I will fix it.

**Pull request merged**